nvtabular/loader/backend.py

```diff
@@ -180,12 +180,15 @@ def __init__(
         global_size=None,
         global_rank=None,
         drop_last=False,
+        sparse_list=None,
+        sparse_max=None,
```

Comment:
We might want to add a docstring to the `DataLoader` class to explain what all these params mean sometime soon.

Reply:
they are in the framework specific dataloaders, if we want to transition to the backend... we can do that...

Reply:
Eh, maybe not worth it.